source/operations/data-recovery/recover-after-site-failure.rst

```diff
@@ -1,13 +1,95 @@
+2. Configure the new site with the same Identity Provider (IDp) as the other sites
+3. :ref:`Expand the existing site replication <minio-expand-site-replication>` by adding the newly deployed site
+4. Remove the lost site from the site replication configuration
+
+Site replication healing automatically adds IAM settings, buckets, bucket configurations, and objects from the existing site(s) to the new site with no further action required.
+
+If you are switching from using bucket replication to using site replication, you must first remove all bucket replication rules from the healthy site prior to setting up site replication.
+
+Replication Resync
+------------------
+
+For scenarios where :ref:`bucket replication <minio-bucket-replication>` was in place prior to the failure, you can use :mc-cmd:`mc replicate resync` to restore data to a new site.
+Create a new site to replace the failed deployment, then synchronize the data from an existing bucket to the new site.
+
+1. Deploy a new MinIO site
+2. Set up IAM and users as needed
+3. On the site with data, create a new ``remote target`` using the :mc-cmd:`mc admin bucket remote add` command and record the ARN from the output
+4. From the site with the data, use the :mc-cmd:`mc replicate resync` command with the ARN from the previous command to rebuild the on the new site
+5. Wait for re-synchronization to complete (us :mc-cmd:`mc replicate resync status` to check)
+6. Set up bucket replication rule(s) from the new MinIO site to the existing target bucket(s)
+7. `(Optional)` Delete the bucket replication rules from the target deployment(s) to restore an active-passive replication scenario
+
+Bucket Replication
+------------------
+
+:ref:`Bucket replication <minio-bucket-replication>` can directly restore the site contents by performing a replication from the target bucket(s) to a new MinIO site.
+
+As a passive process, bucket replication may not perform as quickly as desired for a site recovery scenario.
+In such cases where performance is needed, use :mc-cmd:`mc replicate resync` described above instead.
+
+Bucket replication rules copy the object, its version ID, versions, and other metadata to the target bucket.
+MinIO can restore the object with all of these attributes to a new MinIO site if bucket replication had already been in use prior to the site loss.
+
+1. Deploy a new MinIO site
+2. Set up IAM and users as needed
+3. On the remaining target bucket deployment(s), create bucket replication rule(s) for each bucket to the new MinIO site
+4. Wait for replication to complete
+5. Set up bucket replication rule(s) from the new MinIO site to the existing target bucket(s)
+6. `(Optional)` Delete the bucket replication rules from the target deployment(s) to restore an active-passive replication scenario
+
+   If you prefer to keep an active-active replication between the buckets, do not delete the rules on the target bucket.
```

Comment:
If it's a new site, these wont exist anyways. It's more I think that if they previously had an active-active configuration, they will need to re-create those rules on the new site.

Reply:
This is a second paragraph under step 6. I reworked the sentence to hopefully make that more clear.